frontend/src/pages/Policy/policyContent/HowToUseTheDataContent.tsx

```diff
@@ -0,0 +1,209 @@
+import { METRIC_CONFIG } from '../../../data/config/MetricConfig'
+import { RACE } from '../../../data/utils/Constants'
+import { Fips } from '../../../data/utils/Fips'
+import Custom100kBarChart from '../../../reports/Custom100kBarChart'
+import CustomBreakdownSummary from '../../../reports/CustomBreakdownSummary'
+import CustomChoroplethMap from '../../../reports/CustomChoroplethMap'
+import CustomDisparityBarChart from '../../../reports/CustomDisparityBarChart'
+import CustomRateTrendsLineChart from '../../../reports/CustomRateTrendsLineChart'
+import CustomShareTrendsLineChart from '../../../reports/CustomShareTrendsLineChart'
+import CustomUnknownMap from '../../../reports/CustomUnknownMap'
+
+export const dataVisuals = [
+	{
+		title: 'Rate Choropleth Map',
+		description: 'The rate map shows the geographic distribution of gun violence across the United States. Choropleth maps show the racial/ethnic group with the highest disparity.',
+        sectionId: '#rate-choropleth-map',
+		details: {
+			demographicGranularities: ['Race/ethnicity', 'sex', 'age'],
+			geographicLevels: ['National', 'states', 'counties', 'territories'],
+			alternateBreakdowns: ['Multi-maps', 'State/territory rate extremes (lowest, highest)'],
+			howToUse: [
+				{
+					step: 'Navigate',
+					description: 'Access the rate map on the Health Equity Tracker.'
+				},
+				{
+					step: 'Interpret',
+					description: 'Areas with higher rates are typically highlighted with darker shades. Identify these hotspots.'
+				},
+				{
+					step: 'Advocate',
+					description: 'Present this map to local officials to show where interventions are most needed.'
+				}
+			]
+		},
+		customCard: <CustomChoroplethMap/>,
+	},
+	{
+		title: 'Rates Over Time Chart',
+		description: 'This visualization tracks the changes in gun violence rates over specified periods.',
+        sectionId: '#rates-over-time-chart',
+		details: {
+			demographicGranularities: ['Race/ethnicity', 'sex', 'age'],
```

Comment:
Same, it's the same demographics for every card, so extract that to the top of the file. also include urbanicty